database/seeds/UsersTableSeeder.php

```diff
@@ -18,14 +18,17 @@ public function run()
             'permission' => 1,
             'verified' => true
         ]);
+        factory(App\MacAddress::class, 3)->create(['user_id' => 1]);
```

Comment:
I don't know what does the `factory` :D
Why are those the parameters for it?

Reply:
Generates dummy data from the given class. The number is the rows created.
It also does it in a clever manner, if you migrate:refresh, you can find users with names that can fail some assertions.